x-pack/plugins/security_solution/common/risk_engine/constants.ts

```diff
@@ -6,3 +6,15 @@
  */
 
 export const MAX_SPACES_COUNT = 1;
+
+export const KIBANA_SAVED_OBJECT_MANAGE_ALL_PRIVILEGE = 'feature_savedObjectsManagement.all';
+export const RISK_ENGINE_REQUIRED_KIBANA_PRIVILEGES = [KIBANA_SAVED_OBJECT_MANAGE_ALL_PRIVILEGE];
```

Comment:
Isn't the `feature_siem.entity-analytics` kibana privilege also needed to use entity analytics as a whole? 
cc @machadoum

Reply:
Well, I just noticed this privilege is checked using the route API _access_ tag [here](https://github.com/elastic/kibana/pull/171250/files#diff-b3289076f7fa8177f20208f872c59ecf1327e67aef13da4cdd4a7529915c432cR26), so this new `/privileges` route will return "forbidden" if the user does not have the `entity-analytics` privilege. 
Does it make sense? 🤔 Just asking, maybe it does.

Reply:
> this new /privileges route will return "forbidden" if the user does not have the entity-analytics privilege.
Does it make sense?

Hmmm I am not 100% sure here. Thinking about it, the user will only make that API call if they are in the entity analytics management page in Kibana, so they have no need for the API if they don't have that permission. So maybe it is better to keep it?

Reply:
Okay. Yes, this is an internal route, if it is only called from a page where the user needs the `entity-analytics` privilege to access, it makes sense 👍
resolving and approving